### PR TITLE
Add graphql cops

### PR DESCRIPTION
**Pull request by Anastastasia-B**
Added GraphQL cops from https://github.com/DmitryTsepelev/rubocop-graphql

Before you submit a pull request, please make sure you have to follow:

- [x] read and know items from the [Contributing Guide](https://github.com/datarockets/datarockets-style/blob/master/CONTRIBUTING.md#pull-requests)
- [x] add a description of the problem you're trying to solve (short summary from related issue)
- [x] verified that cops are ordered by alphabet
- [x] add a note to the style guide docs (if it needs)
- [x] add a note to the changelog file
- [ ] the commit message contains the number of the related issue (if it presents)
  and word `Fix` if this PR closes related issue
- [x] squash all commits before submitting to review

### CODE REVIEW

File: datarockets-style.gemspec [hunk 1]
```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "rubocop", "~> 1.45.1"
+  spec.add_dependency "rubocop-graphql", "~> 1.1.1"
```
Comment (Anastastasia-B) [inline datarockets-style.gemspec:36]: Can not use later versions because they depend on a later version of rubocop

File: config/graphql.yml
Comment (Anastastasia-B) [inline config/graphql.yml:1]: Disabled ExtractInputType and ExtractType as they are proven not to bringing any positive value & set ResolverMethodLength to 10 - same as any method max lenght. Later the cops can be re-configured accordingly to the team’ needs

File: config/graphql.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require: rubocop-graphql
+
+GraphQL/ExtractInputType:
+  Enabled: false
```
Comment (roman-dubrovsky) [inline config/graphql.yml:4]: just add a empty line

```suggestion
  Enabled: false
  
```

File: datarockets-style.gemspec (hunk 1 above)
Comment (roman-dubrovsky) [reply datarockets-style.gemspec:36]: @Anastastasia-B I'll upgrade rubocop before pushing the new version, so don't worry 😉

Comment (Anastastasia-B) [reply datarockets-style.gemspec:36]: Turned out that versions after 1.1.1 also depend on Ruby 3.0 so still can not use later versions

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 
 ## master
 
+### Added
```
Comment (an-korn) [inline CHANGELOG.md:7]: Don't forget to add the `release` header here, see https://guides.rubygems.org/patterns/#semantic-versioning to choose what's the version you should use for this release

**Review by an-korn — APPROVED** [review]
LGTM with comments 👍

**Review by atatarenko — APPROVED** [review]
LGTM!

File: doc/STYLE_GUIDE_GRAPHQL.md [hunk 4]
```diff
@@ -0,0 +1,486 @@
...
+  end
+
+  def last_name
+    object.contact_data.last_name
+  end
+end
+```
+
+* <a name="graphql-field-description"></a>
+  All field definitions should be grouped together.
```
[the remark this answers is not in the document]
Comment (Anastastasia-B) [reply doc/STYLE_GUIDE_GRAPHQL.md]: Nice catch, tahnk you!

**Review by roman-dubrovsky — APPROVED** [review]

**Pull request merged**